aks/main.tf

```diff
@@ -10,7 +10,6 @@ resource "azurerm_kubernetes_cluster" "k8s" {
     location            = "${var.location}"
     resource_group_name = "${var.az_resource_group}"
     dns_prefix          = "${var.dns_prefix}"
-    kubernetes_version  = "${var.k8s_version}"
```

Comment:
should we still keep k8s version configurable with default value being set to what was tested. We can also surface that in docs as recommended version.